backend/stylists/app/Http/Controllers/CollectionController.php

```diff
@@ -33,17 +41,63 @@ public function index(Request $request, $action, $id = null, $action_id = null)
         return $this->$method($request);
     }
 
-    public function getList(Request $request){
+    public function getList(Request $request)
+    {
+        $this->base_table = 'collections';
+        $this->initWhereConditions($request);
+        $this->initFilters();
+        
+        $view_properties = array(
+            'stylists' => $this->createdBy,
+            'statuses' => $this->statuses,
+            'genders' => $this->genders,
+            'occasions' => $this->occasions,
+            'body_types' => $this->body_types,
+            'budgets' => $this->budgets,
+           'age_groups' => $this->age_groups
+        );
+        
+        foreach ($this->filter_ids as $filter) {
+            $view_properties[$filter] = $request->has($filter) && $request->input($filter) !== "" ? intval($request->input($filter)) : "";
+        }
+        $view_properties['stylist_id'] = $request->has('stylist_id') && $request->input('stylist_id') !== "" ? intval($request->input('stylist_id')) : "";
+        
+        $view_properties['nav_tab_index'] = '0';
+        $user_data = Auth::user();
+        $entity_nav_tabs = array(
+            EntityType::CLIENT
+        );
+
+        $view_properties['entity_type_names']= array(
+            EntityTypeName::CLIENT
+        );
+        
         $paginate_qs = $request->query();
         unset($paginate_qs['page']);
 
         $collections =
             Collection::with('gender','status','body_type','budget','occasion','age_group')
+                ->where($this->where_conditions)
                 ->orderBy('id', 'desc')
                 ->simplePaginate($this->records_per_page)
                 ->appends($paginate_qs);
 
         $view_properties['collections'] = $collections;
+        $view_properties['search']      = $request->input('search');
+        $view_properties['exact_word']  = $request->input('exact_word');
+        $view_properties['from_date']   = $request->input('from_date');
+        $view_properties['to_date']     = $request->input('to_date');
+
+        $view_properties['min_price']   = $request->input('min_price');
+        $view_properties['max_price']   = $request->input('max_price');
+        
+        $view_properties['app_sections']            = AppSections::all();
+        $view_properties['logged_in_stylist_id']    = $user_data->id;
+        $view_properties['popup_entity_type_ids']   = $entity_nav_tabs;
+        $view_properties['entity_type_to_send']     = EntityType::LOOK;
```

Comment:
From this controller 'entity_type_to_send' should be Collection so this line should be like
$view_properties['entity_type_to_send']     = EntityType::COLLECTION;